lib/ts/Diags.cc

```diff
@@ -226,43 +214,50 @@ void
 Diags::print_va(const char *debug_tag, DiagsLevel diags_level, const SourceLocation *loc, const char *format_string,
                 va_list ap) const
 {
-  struct timeval tp;
-  const char *s;
-  char *buffer, *d, timestamp_buf[48];
-  char format_buf[1024], format_buf_w_ts[1024], *end_of_format;
-
   ink_release_assert(diags_level < DiagsLevel_Count);
 
-  ////////////////////////////////////////////////////////////////////////
-  // there are 2 format buffers that hold a printf-style format string  //
-  // format_buf contains <prefix_string>: (<debug_tag>) <format_string> //
-  // and format_buf_w_ts has the same thing with a prepended timestamp. //
-  ////////////////////////////////////////////////////////////////////////
+  const size_t BUF_CAPACITY = 1024;
```

Comment:
Is this used anywhere else? If not, might as well just put the `1024` explicitly in the `LocalBufferWriter` declaration.

Reply:
Yes, we can clean this up. Formerly there were two buffers, and now there is one.